client/package.json

```diff
@@ -42,11 +43,12 @@
     "mirror-creator": "1.1.0",
     "mkdirp": "^0.5.1",
     "moment": "^2.17.1",
+    "moment-timezone": "^0.5.11",
     "node-sass": "^4.5.0",
     "react": "^0.14.8 || ^15.0.0",
     "react-ace": "^4.1.1",
     "react-dom": "^0.14.8 || ^15.0.0",
-    "react-intl": "^2.1.5",
+    "react-intl": "jeremyyap/react-intl#build",
```

Comment:
Maybe I will fork the other guy's repo and make the requested changes.

There are multiple ways to add comments to JSON files, but they don't seem very neat or proper. http://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install